lib/rubocop/cli.rb

```diff
@@ -104,7 +104,7 @@ def apply_default_formatter
     def print_available_cops
       # Load the configs so the require()s are done for custom cops
       @config_store.for(Dir.pwd)
-      cops = Cop::Cop.all
+      cops = Cop::Cop.cop_store
```

Comment:
This looks a bit odd - I would not expect something named cops to be an instance of `cop_store`.